packages/near-membrane-base/src/membrane.ts

```diff
@@ -1726,6 +1803,33 @@ export function createMembraneMarshall() {
                     } catch (e: any) {
                         throw pushErrorAcrossBoundary(e);
                     }
+                    if (
+                        result &&
+                        configurable === false &&
+                        typeof foreignCallableNonConfigurableDescriptorCallback === 'function'
+                    ) {
+                        let unsafeDesc;
+                        try {
+                            unsafeDesc = ReflectGetOwnPropertyDescriptor(target, key);
+                        } catch (e: any) {
+                            throw pushErrorAcrossBoundary(e);
+                        }
+                        if (unsafeDesc) {
+                            const descMeta = getDescriptorMeta(unsafeDesc);
+                            const { 0: descMeta0 } = descMeta;
+                            if (descMeta0 === false) {
```

Comment:
Any reason not to name this: `descMeta0` => `configurable`?

Reply:
I don't wanna shadow the `configurable` in the upper scope.